domains/amang.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "G-Aman",
+    "email": "mail@amang.pp.ua"
+  },
+  "records": {
+    "CNAME": "g-aman.github.io",
+    "MX": "mx.amang.in"
+  }
```

```suggestion
  "records": {
    "CNAME": "g-aman.github.io",
    "MX": "mx.amang.in"
  },
  "proxied": true
```

You can't combine CNAME with other records unless it was proxied

---

Commit Suggestion